certified-connectors/Shopranos/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "publisher": "SoftOne Technologies S.A",
+    "stackOwner": "SoftOne Technologies S.A",
```

Comment:
Publisher and Stackowner are different, please update as per the format in this link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json